lnbits/tasks.py

```diff
@@ -133,8 +133,14 @@ async def check_pending_payments():
     the backend and also to delete expired invoices. Incoming payments will be
     checked only once, outgoing pending payments will be checked regularly.
     """
+    sleep_time = 60 * 30  # 30 minutes
 
     while settings.lnbits_running:
+        funding_source = get_funding_source()
+        if funding_source.__class__.__name__ == "VoidWallet":
+            logger.warning("Task: skipping pending check for VoidWallet")
+            await asyncio.sleep(sleep_time)
+            continue
```

Comment:
Why not exit the loop and let the task complete. There is no way to switch from `VoidWallet` to a real wallet.
```suggestion
            return
```